ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-sync/src/main/java/ch/admin/bag/covidcertificate/backend/verifier/sync/syncer/DscUploadClient.java

```diff
@@ -58,22 +58,24 @@ public DscUploadResponse uploadDscs()
 
         List<CertToUpload> certsToUpload = certUploadDataService.findCertsToUpload();
         logger.info(
-                "found {} dscs to upload. aliases: {}",
+                "found {} dscs to upload. aliases/slots: {}/{}",
```

Comment:
instead of logging a list of aliases and the list of slots i would log the alias/slot combination for each cert to upload.

``` suggestion
      logger.info(
                "found {} dscs to upload. aliases/slots: {}",
                certsToUpload.size(),
                certsToUpload.stream()
                        .map(c -> c.getAlias() + "/" + c.getSlot())
                        .collect(Collectors.toList()));
```